utilities/pipelines/e2eValidation/resourceRemoval/helper/Invoke-ResourceRemoval.ps1

```diff
@@ -156,6 +156,7 @@ function Invoke-ResourceRemoval {
 
             $resourceGroupName = $ResourceId.Split('/')[4]
             $resourceName = Split-Path $ResourceId -Leaf
+            $subscriptionId = $ResourceId.Split('/')[2]
```

Comment:
Intended?

Reply:
Yes, but moved to PR: https://github.com/Azure/bicep-registry-modules/pull/4047

Reply:
Can this be resolved? :)